omnibus/config/software/oracledb-py3.rb

```diff
@@ -16,6 +16,7 @@
   license_file "./LICENSE.txt"
 
   command "sed -i 's/cython/cython<3.0.0/g' pyproject.toml"
+  command "sed -i 's/cryptography>=3.2.1/cryptography>=3.2.1,<42.0.0/g' setup.cfg"
 
   command "#{install_dir}/embedded/bin/pip3 install ."
```

Comment:
I think this would be bit safer:

```suggestion
  command "#{install_dir}/embedded/bin/pip3 install cryptography<42.0.0 ."
```

Reply:
Can you explain why? I was just trying to be consistent with the other sed that is already in this file.

Reply:
The patching via `sed` in the first case is needed because it modifies a build dependency.

My reasoning is that a `sed` substitution (as with any patch) is more "coupled" to what we're modifying; changes to `setup.cfg` could more easily break the patch than adding the constraint on `install`.

Also, that's what I've done myself in the past in these situations.